src/e2e/java/teammates/e2e/cases/lnp/InstructorStudentEnrollmentLNPTest.java

```diff
@@ -108,25 +110,25 @@ protected LNPTestData getTestData() {
                     csvRow.add(instructor.courseId);
 
                     // Create and add student enrollment data with a team number corresponding to each section number
-                    int dataLength = 75 * NUM_STUDENTS_PER_INSTRUCTOR;
-                    StringBuilder enrollData = new StringBuilder(dataLength);
-                    enrollData.append("\"Section|Team|Name|Email|Comments\n");
+                    List<StudentsEnrollRequest.StudentEnrollRequest> enrollRequests = new ArrayList<>();
 
                     for (int i = 0; i < NUM_STUDENTS_PER_INSTRUCTOR; i++) {
+
                         String name = instructor.name + ".Student" + i;
                         String email = instructor.name + ".Student" + i + "@gmail.tmt";
+                        String team = String.valueOf(i / NUM_STUDENTS_PER_SECTION);
+                        String section = String.valueOf(i / NUM_STUDENTS_PER_SECTION);
+                        String comment = "no comment";
 
-                        enrollData.append(i / NUM_STUDENTS_PER_SECTION)
-                                .append('|')
-                                .append(i / NUM_STUDENTS_PER_SECTION)
-                                .append('|')
-                                .append(name)
-                                .append('|')
-                                .append(email)
-                                .append("|no comment\n");
+                        enrollRequests.add(
+                                new StudentsEnrollRequest.StudentEnrollRequest(name, email, team, section, comment));
                     }
-                    enrollData.append('\"');
-                    csvRow.add(enrollData.toString());
+                    StudentsEnrollRequest requestBody = new StudentsEnrollRequest(enrollRequests);
+
+                    // Sanitize the data for CSV entry.
+                    String enrollData = String.format("\"%s\"",
+                            JsonUtils.toJson(requestBody).replace("\n", "").replace("\"", "\"\""));
```

Comment:
Let's extract this out somewhere? This operation will be necessary for all requests which have request body.